kong/plugins/prometheus/exporter.lua

```diff
@@ -99,12 +105,38 @@ local function init()
   if enterprise then
     enterprise.init(prometheus)
   end
+
+
+  local role = kong.configuration.role
```

Comment:
Please pull this up at the module level and replace line 372 with `if role == "control_plane"`. That makes the code a bit more readable.